lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -94,6 +95,15 @@ def self.increase_version(current_version, type_of_bump, snapshot)
         end
       end
 
+      private_class_method def self.latest_non_prerelease_version_number
+        Actions
+          .sh("git tag", log: false)
```

Comment:
Added the `log: false` to not output all the tags in the console while doing this, since there can be a lot

Reply:
beautiful